server/functions.lua

```diff
@@ -609,3 +609,51 @@ function QBCore.Functions.PrepForSQL(source, data, pattern)
     end
     return true
 end
+
+---Do a money transactio between players
+---@param emitercid string
+---@param emitermoneytype string -- Only money types in QBConfig.Money.MoneyTypes
+---@param receivercid string
+---@param receivermoneytype string -- Only money types in QBConfig.Money.MoneyTypes
+---@param quant number
+---@param reason string
+---@return boolean
+function QBCore.Functions.TransferMoney(emitercid, emitermoneytype, receivercid, receivermoneytype, quant, reason)
+    local EmiterPlayer = QBCore.Functions.GetPlayerByCitizenId(emitercid)
+    local ReceiverPlayer = QBCore.Functions.GetPlayerByCitizenId(receivercid)
+    if not tonumber(quant) then return false end
+    quant = tonumber(quant) or 0
```

Comment:
operation with 0 should not be done I think. Probably you can do `if not tonumber(quant) or  tonumber(quant) <= 0 then return false end` or just `if tonumber(quant) <= 0 then return true end`